src/sentry/static/sentry/app/views/groupActivity/noteInput.jsx

```diff
@@ -19,10 +35,24 @@ var NoteInput = React.createClass({
       expanded: false,
       preview: false,
       updating: updating,
-      value: updating ? item.data.text : ''
+      value: defaultText
     };
   },
 
+  componentWillUpdate(nextProps, nextState) {
```

Comment:
Is there a convention to say, ignore some arguments? We don't need `nextProps`, and in Python, I'd do `_`.